multiple changes to make document easier to read (#23842)
grafana/grafana: docs/sources/features/datasources/loki.md

* Add fields parsed from the log message. 
* Add a link that uses the value of the field. 

> You can use this functionality to link your tracing backend directly from your logs, or link a user profile page if a userId is present in the log line. These links appear in the [log details](/features/explore/#labels-and-parsed-fields).

Reviewer: ```suggestion
You can use this functionality to link to your tracing backend directly from your logs, or link to a user profile page if a userId is present in the log line. These links appear in the [log details](/features/explore/#labels-and-parsed-fields).
```

> You can use this functionality to link your tracing backend directly from your logs, or link a user profile page if a userId is present in the log line. These links will be shown in the [log details](/features/explore/#labels-and-parsed-fields).

Reviewer: I think this change is wrong, the original wording should be kept.

{{< docs-imagebox img="/img/docs/v65/loki_derived_fields.png" class="docs-image--no-shadow" caption="Screenshot of the derived fields configuration" >}}
Each derived field consists of:

- **Name:** Shown in the log details as a label.

> - **Regex:** A Regex pattern that runs on the log message and captures part of it as the value of the new field. Can only contain pattern to capture a single group.

Reviewer: ```suggestion
- **Regex:** A Regex pattern that runs on the log message and captures part of it as the value of the new field. Can only contain a single capture group.
```

The [same rules that apply for Prometheus Label Selectors](https://prometheus.io/docs/prometheus/latest/querying/basics/#instant-vector-selectors) apply for Loki Log Stream Selectors.

> Another way to add a label selector, is in the table section, clicking on the **Filter** button beside a label will add the label to the query expression. This even works for multiple queries and will add the label selector to each query.

Reviewer 2: ```suggestion
Another way to add a label selector is in the table section. Click **Filter** beside a label to add the label to the query expression. This even works for multiple queries and will add the label selector to each query.
```
